m4_sessions/1710-inning/group_five.md

```diff
@@ -1,28 +1,62 @@
+
+A: Brief intro of what it is, why it’s useful  
+B: Fist to 5 about how comfortable you are with it  
+C: Other icebreaker?  
+
+**2. Jimmy** - Discussion on why ppl feel comfortable or uncomfortable ~ 10 min  
+
+Goal: Encourage participants to have an open conversation 
+
+A: Open up with some ‘dig deeper’ questions
+  - What are your concerns or worries about cold outreach?
+  - What do you think the value is of cold outreach?  
+  
+B: Encourage to think about how each participant may have already been successful in the past or when they failed and strategies to move past it 
+
+C: Follow up with positive reinforcement / cat/ dog pictures :) (e.g. it’s okay to be intimidated and can still do a good job at it)  
+
+**3. Ellen** - General/different strategies/steps/expectations to use for upcoming small group workshop sessions ~ 5 mins  
+
+Goal: Discuss as a group or present a slide with ‘best practices’ or our personal approach/what is successful for different personalities  
+
+**4. Live group demo: Katie + participant**  ~ 25 min  
+
+Goal: Participant presents a person/company/position of interest and one of the presenters guides them through the process of finding and messaging the ‘best’ person in an open style discussion. Message is sent at the end of demo.  
+
+Note: We should encourage group members to write down questions during the demo and follow up during the small group work time, in order to make sure this doesn’t side track the goal of this part of the session -- to complete a search and send a message successfully.  
+
+**5. Small group workshop:** large group breaks into 4 (do a 1-4 count) and each of us facilitate so that each group member repeats the steps that were just demonstrated and implements some of the tools discussed ~ 25 min  
+
+Goal:  Each participant researches and finds a person, and sends a message today. Facilitators make sure everyone is on track, no one gets stuck by checking in and sitting with their group. Available to help search and to help phrase messages if needed.
+
+**6. Conclusion: Katie** -- Come back as a large group ~10 min  
 
-Session Outline:  Include the flow of the session and the time for each section.
-1. Brief intro of what it is, why it useful
-2. fist to 5 about how comfortable you are with it
-3. discussion on why ppl feel comfortable or uncomfortable
-4. general strategies/steps/expectations
-5. live group demo
-6. 3 small groups - work on your own cold outreach
-7. large group - discuss issues that came up
+A: Good job everyone!  
+B: Surprises? What was comfortable/uncomfortable?  
+ - How do you feel about cold outreach now?
+ - What concerns do you still have?  
 
+C: Take aways, can you do this again?  
 
+Slideshow: https://docs.google.com/presentation/d/1lYv1VyxR9kwjmzaEABE0LKz32GZm-INSoT7iWEpRD08/edit?usp=sharing
```

Comment:
Everything about this session is on point.  I sent it to Allison so she could see it and she is so impressed!  Excellent job of clarifying who is doing what and what you can do as a group.  I can't wait for this!!

Reply:
@emcooper @DesTodo @jtruong2 We did good!